src/nvim/api/vim.c

```diff
@@ -1679,21 +1465,6 @@ void nvim_del_keymap(uint64_t channel_id, String mode, String lhs, Error *err)
   nvim_buf_del_keymap(channel_id, -1, mode, lhs, err);
 }
 
-/// Gets a map of global (non-buffer-local) Ex commands.
-///
-/// Currently only |user-commands| are supported, not builtin Ex commands.
-///
-/// @param  opts  Optional parameters. Currently only supports
-///               {"builtin":false}
-/// @param[out]  err   Error details, if any.
-///
-/// @returns Map of maps describing commands.
-Dictionary nvim_get_commands(Dict(get_commands) *opts, Error *err)
```

Comment:
if we really don't like this in vim.c, how about renaming vimscript.c to editor.c and put most of the Ex/func stuff there?